travis/build-crossdock.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+make crossdock
+
+export REPO=jaegertracing/xdock-go
+export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+
+# Only push the docker container to Docker Hub for master branch
+if [ "$BRANCH" == "master" ]; then echo 'upload to Docker Hub'; else echo 'skip docker upload for PR'; exit 0; fi
+
+docker build -f crossdock/Dockerfile -t $REPO:$COMMIT .
```

Comment:
nit: it would make sense to do `set -x` before the build, in case it fails. You can do `docker login` right after L11